packaging/nix/override.nix

```diff
@@ -0,0 +1,50 @@
+# Override function for umu-launcher-unwrapped
+pkgs: prevPkgs: finalAttrs: prevAttrs:
+let
+  inherit (pkgs)
+    lib
+    cargo
+    zstd
+    ;
+  inherit (pkgs.python3Packages)
+    cbor2
+    truststore
+    xxhash
+    ;
+  inherit (pkgs.rustPlatform)
+    cargoSetupHook
+    importCargoLock
+    ;
```

Comment:
Can we please use alejandra? I thought it was specified as the formatter in a previous PR as well.

Reply:
No preference from me

Reply:
Would also prefer a formatter to be specified don't really have a preference for what. Would be out of scope for this pr though.

Reply:
#350